test/test_individual.py

```diff
@@ -3,21 +3,46 @@
 import pytest
 
 import cgp
-from cgp.individual import IndividualSingleGenome
+from cgp import Genome, IndividualSingleGenome, IndividualMultiGenome
 from cgp.genome import ID_INPUT_NODE, ID_OUTPUT_NODE, ID_NON_CODING_GENE
 
 
-def test_pickle_individual():
+def _create_individual(fitness, genome, ind_type="single"):
```

Comment:
maybe be extra verbose and always write `individual` instead of `ind`?

Reply:
okay, implemented.

Reply:
9e92d2814a90543959b922660f462987ddb538fc